### DISCUSSION TITLE
Nextjs 13 Not found page is keep reloading every 3 seconds

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [vercel/next.js#50429] [App Router]
![image](https://github.com/vercel/next.js/assets/44552983/425f8194-7b3b-49a4-ae30-ec101768ff10)

```tsx
import Link from 'next/link';

export default function NotFound() {
  return (
    <div>
      <h2>Not Found</h2>
      <p>Could not find requested resource</p>
      <p>
        View <Link href="/blog">all posts</Link>
      </p>
    </div>
  );
}

```

@leerob Could you please help me ?

**Comment by icyJoseph** [comment]
Hi,

My take is that, this is only for the development mode. It is done so that it can catch up on a path that's newly created. I think there's even a test for it.

The `_error` page in pages dir did something similar, although it didn't have to refresh the entire page. Perhaps it shouldn't need to refresh the page?

**Reply by MuttakinHasib** [reply]
How can I prevent this behavior in the development mode?

**Comment by abdellatif-temsamani** [comment]
i have the same problem

**Comment by Script-Sage28** [comment]
It's reloading the page every 3 seconds. What I did was, inside the app directory, I created a folder named `[...not-found]` and then inside it, there's a `page.jsx` that contains the custom 404 page.

It worked and didn't reload the page every 3 seconds.

**Reply by SinniS1** [reply]
![image](https://github.com/vercel/next.js/assets/117257605/cd698ca8-6b0e-46e1-8a46-6ebd423425f9)
![image](https://github.com/vercel/next.js/assets/117257605/4562bbfe-8d27-472d-a043-785768385fb5)

Bro 😭  
please help

**Reply by Script-Sage28** [reply]
Maybe you haven't created the `not-found.tsx` file since it's missing.
